src/main/java/org/folio/rest/validator/ValidatorUtil.java

```diff
@@ -111,4 +129,18 @@ public static boolean isUrlValid(String url) {
       return false;
     }
   }
+
+  public static void checkUrlFormat(String paramName, String value) {
+    if(!StringUtils.startsWith(value, "http://") && !StringUtils.startsWith(value, "https://")) {
```

Comment:
There is already isUrlValid method, so this can be refactored to "if(!isUrlValid(value))"